sdk/storage/azure-storage-common/src/file_io.cpp

```diff
@@ -22,9 +26,11 @@ namespace Azure { namespace Storage { namespace _internal {
 #if defined(AZ_PLATFORM_WINDOWS)
   FileReader::FileReader(const std::string& filename)
   {
+    HANDLE fileHandle = NULL;
```

Comment:
I'm not sure if this is the right way to initialize handle

```suggestion
    HANDLE fileHandle = INVALID_HANDLE_VALUE;
```

or just

```suggestion
    HANDLE fileHandle;
```